pallets/exchange/src/direct.rs

```diff
@@ -260,9 +261,10 @@ impl<'a, T: Config> DirectTradeData<'a, T> {
 	}
 
 	/// Send pallet event after a fee is transferred.
-	fn send_trade_fee_event(from: &T::AccountId, to: &T::AccountId, asset: AssetId, amount: Balance) {
+	fn send_trade_fee_event(from: &T::AccountId, intention: &Intention<T>, to: &T::AccountId, asset: AssetId, amount: Balance) {
```

Comment:
Do we only send fee from one side?

Reply:
@enthusiastmartin

Reply:
it is sent for each intention/transaction. you can see it in the tests.